### PR TITLE
New node Element.GetGeometry

### PR DESCRIPTION
**Pull request by dnenov**
### Purpose

Part of the new nodes PRs, this small PR adds a new Element.GetGeometry node, which incorporates an input for Detail Level.

getgeometry

- added new node Element.GetGeometry that has detail level
- 
### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] Snapshot of UI changes, if any.

### Reviewers

@reddyashish 
@Amoursol

### CODE REVIEW

File: src/Libraries/RevitNodes/Elements/Element.cs [hunk 1]
```diff
@@ -585,6 +586,56 @@ public object[] Geometry()
             return converted.ToArray();
         }
 
+        /// <summary>
+        /// Returns all geometry associated with an element. Ignores transforms when used with linked elements.
+        /// <param name="detailLevel">Detail level</param>
+        /// <returns>List of geometry from the element</returns>
+        /// </summary>
+        public object[] GetGeometry(string detailLevel = "Medium")
```
Comment (mjkkirschner) [inline src/Libraries/RevitNodes/Elements/Element.cs:594]: doesn't an enum make more sense here? What about localized envs? How will users know what strings to use etc? I think there are other examples in the repo - maybe?

**Comment by mjkkirschner** [comment]
wheres the tests ;)?

File: src/Libraries/RevitNodes/Elements/Element.cs (hunk 1 above)
Comment (dnenov) [reply src/Libraries/RevitNodes/Elements/Element.cs:594]: It actually works with the current `Detail Level` dropdown node in Dynamo which yields a string output. It looks like this:

image

The output of the `Detail Level` node is not language dependent, so it made sense to do it this way. Let me know if that still bothers you though!

**Comment by dnenov** [comment]
Recreated to master branch https://github.com/DynamoDS/DynamoRevit/pull/2924, closing this PR.

**Pull request closed without merging**